src/app/components/chat-interface/ChatInterface.tsx

```diff
@@ -35,31 +38,76 @@ export default function ChatInterface({
   }
 
   return (
-    <div className="flex flex-col justify-end gap-y-16 px-2 py-4">
-      <Chats chats={chats} />
-      <PromptForm onPrompt={handlePrompt} history={chats} />
-    </div>
+    <Container maxWidth="lg" sx={{ height: '100vh' }}>
```

Comment:
Removes the second scrollbar
```suggestion
    <Container>
```